src/main/java/org/jenkinsci/plugins/durabletask/BourneShellScript.java

```diff
@@ -137,74 +146,120 @@ public String getScript() {
         }
 
         String scriptPath = shf.getRemote();
-        List<String> args = new ArrayList<>();
-
-        if (os != OsType.DARWIN) { // JENKINS-25848
-            args.add("nohup");
```

Comment:
Do we need to preserve this in some form for the fallback method? It doesn't look like it was moved somewhere else.

Reply:
Done in ef70f1891.